canarytokens/channel_input_wireguard.py

```diff
@@ -20,7 +20,7 @@
 from canarytokens.settings import SwitchboardSettings
 from canarytokens.switchboard import Switchboard
 from canarytokens.tokens import Canarytoken
-from canarytokens.exceptions import NoCanarytokenFound, NoCanarytokenPresent
+from canarytokens.exceptions import NoCanarytokenFoundInQuery
```

Comment:
Is it not simpler to have 
`NoCanarytokenFound` and `NoCanarydropFound` 
than `NoCanarytokenFoundInQuery` and `NoCanarytokenPresent`
?

Reply:
@wleightond: thanks! I like that.